src/developer/web-api/new-tracker.md

```diff
@@ -1276,6 +1276,7 @@ The endpoint returns a list of tracked entities that match the request parameter
 |`includeDeleted`|`Boolean`|`true`&#124;`false`|Indicates whether to include soft-deleted elements|
 |`includeAllAttributes`|`Boolean`|`true`&#124;`false`|Indicates whether to include all TEI attributes|
 |`attachment`|`String`| |The file name in case of exporting as a file|
+|`potentialDuplicate`|`Boolean`|`true`&#124;`false`| Filter the result based on the fact that a tei is a Potential Duplicate. If omitted, we don't check whether a tei is a potentialDuplicate or not. |
```

Comment:
It's not entirely clear to me, from the text, what is the bahaviour:

What is the behaviour if the parameter is set to "true"?